apps/src/templates/CountryAutocompleteDropdown.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component, PropTypes } from 'react';
+import VirtualizedSelect from 'react-virtualized-select';
+import 'react-virtualized/styles.css';
+import 'react-select/dist/react-select.css';
+import 'react-virtualized-select/styles.css';
+import i18n from "@cdo/locale";
+import { styles } from './census2017/censusFormStyles';
+import { COUNTRIES } from '../geographyConstants';
+
+export default class CountryAutocompleteDropdown extends Component {
+  static propTypes = {
+    setField: PropTypes.func,
+    showErrorMsg: PropTypes.bool,
+    required: PropTypes.bool,
+    value: PropTypes.string
+  };
+
+  sendToParent = (event) => {
+    this.props.setField("country", event);
+  }
```

Comment:
nit: Why `setField` and `sendToParent`? If this is just (as it appears to be) bubbling up the change event to the parent, why not stick with the more standard names `handleChange`, and `onChange`?